src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+public class Ui {
+
+    /**
+     * Prints out text with decorative lines.
+     * Used for all output by Duke.
+     *
+     * @param str Text to be printed.
+     */
+    public void printOut(String str) {
+        String line = "____________________________________________________________\n";
+        System.out.println(line + str + "\n" + line);
+    }
+
+    /**
+     * Prints out welcome message.
+     */
+    public String showWelcome() {
+        return "Hello! I'm Duke.\n"
+                + "What can I do for you?";
+    }
+
+    /**
+     * Prints out goodbye message.
+     */
+    public String showGoodbye() {
```

Comment:
Is it better to name it getGoodbyeMessage?

Reply:
Nice suggestion! Will make this and the other changes.